lightwood/encoder/categorical/onehot.py

```diff
@@ -77,7 +76,7 @@ def encode(self, column_data):
 
         return torch.Tensor(ret)
 
-    def decode(self, encoded_data):
+    def decode(self, encoded_data, predict_proba=False):
```

Comment:
same as above, the encoder has nothing to do with predictions, what's being done here is something else, asking the encoder to... not decode, and instead just return the data it got + some tiny format mods.